istioctl/cmd/analyze.go

```diff
@@ -120,11 +131,7 @@ istioctl experimental analyze -k -d false
 				return err
 			}
 
-			for _, m := range messages {
-				fmt.Fprintf(cmd.OutOrStdout(), "%v\n", m.String())
-			}
-
-			return nil
+			return handleAnalyzeMessages(cmd.OutOrStdout(), messages)
```

Comment:
One thing to keep in mind: we are going to have more than one reason to affect the return code, not just analysis.

Concrete example I'm working on right now: if we fail to parse any input files, we should return nonzero.